charts/gateway charts/gateway-otk: 10.1.00 release. Java11. Updated major version for the app

GavinWu opened the pull request:
**Description of the change**

10.1.00 Rapier Release which runs on Java11. This is different from 10.0.00 which runs on Java8
docker.io will have 10.1.00
Also increased the major version of the App. 

**Benefits**

Newer release with updated Java and bug fixes within the Layer7 APIM Gateway product

**Drawbacks**

TLSv1.0 and TLSv1.1 is disabled by default due to Java 11.0.11_b9 changes. User can re-enable the disabled algorithm via a change to the java.security file.  

**Applicable issues**

https://techdocs.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-gateway/10-1/release-notes/resolved-issues.html

**Additional information**


**Checklist** <!-- [Place an '[X]' (no spaces) in all applicable fields. Please remove unrelated fields.] -->
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[charts/gateway]`)
- [] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

Gavinswu committed:
10.1.00 release. Java11. Updated major version for the app

GavinWu commented:
The techDoc link will be live the week of 25th July 2021

GavinWu commented:
@gvermeulen7205 : Due to Java11, I changed the apps' major version. Let me know if that is correct. Technically the helm itself isnt a complete backward incompatible. Just the Gateway app itself being incompatible with any existing 10.0.00 Gateways

Gazza7205 commented:
> @gvermeulen7205 : Due to Java11, I changed the apps' major version. Let me know if that is correct. Technically the helm itself isnt a complete backward incompatible. Just the Gateway app itself being incompatible with any existing 10.0.00 Gateways

That sounds good to me - I think a note in the readme 1.x - 2.x covering the Java11 change is needed

GavinWu committed:
Update README.md

GavinWu committed:
Update README.md

GavinWu committed:
Update README.md

GavinWu commented:
> > @gvermeulen7205 : Due to Java11, I changed the apps' major version. Let me know if that is correct. Technically the helm itself isnt a complete backward incompatible. Just the Gateway app itself being incompatible with any existing 10.0.00 Gateways
> 
> That sounds good to me - I think a note in the readme 1.x - 2.x covering the Java11 change is needed

done. Added readme changes to the gateway-otk and gateway chart sections' readme
